.github/workflows/prettify_code.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/marketplace/actions/prettier-action
+# Container action is only supported on Linux
+# styling files with prettier.This includes adding new line.
+
+name: CI
+on:
+  pull_request:
+  push:
```

Comment:
I think it's better to only prettify on pull requests because staging is protected so none will push directly to staging without a pull request
@shakram02 is that correct or you may occasionally make a direct push to staging ?

Reply:
Yes you're right. Pushing to staging directly is restericted

Reply:
Will this change the code being merged? or will just check formatting and fail the PR if the code is not formatted?

Reply:
@amohamed97  Okay 👍
@shakram02      It appends one more commit if code is prettified (Prettified Code!)